docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+
+Once you've imported your contract, the first thing you need to do is compile it. Hit the `Compile` button on the left-hand side. You could also use the keyboard shortcut `Ctrl / Command + S`.
+
+![Remix Compile](intro-to-erc721s/6-remix-compile.png)
+
+Once completed, you will get a green checkmark on the far left tab and will see options to Publish on Ipfs or Swarm. Those are not important to our tutorial. Next, you will click on the bottom tab on the left-hand side to move to the deployment page.
+
+![Remix Deploy Page](intro-to-erc721s/7-deploy-page.png)
+
+Now, we need to change the environment that Remix will try to use to deploy the smart contract. Click on the `Environment` drop-down, and select `Injected web3`.
+
+![Remix Web3](intro-to-erc721s/8-web3.png)
+
+This should prompt you to connect with your MetaMask account. Once connected, you can verify the correct connection by checking that the Account number matches your MetaMask address.
+
+![Remix MetaMask](intro-to-erc721s/9-metamask.png)
+
+Now click on the `Contract` drop-down and select the contract you created and compiled. It should show up with the name you gave it in the Open Zeppelin Wizard.
+
+![Remix Contract](intro-to-erc721s/10-contract.png)
+
+Now, click deploy. This will open MetaMask and ask you to confirm the transaction. Click `Confirm`.
+
+![MetaMask Accept](intro-to-erc721s/11-accept.png)
+
+It may take a second, but once completed, your newly deployed contract will appear underneath the `Transactions Recorded` field.
+
+![Remix Record](intro-to-erc721s/12-transaction-record.png)
+
+Copy your contract’s address and open the [Snowtrace Testnet Explorer](https://testnet.snowtrace.io/). Paste your contract address in the search bar, and click `Search`.
+
+You will now see information on your contract. The first transaction you see should be the contract deployment you just did in the Remix IDE.
+
+![Snowtrace](intro-to-erc721s/13-snowtrace.png)
+
+## Minting an NFT
+
+Now that you have deployed the contract, you can mint the NFT. Go back to the Remix IDE tab and click on your contract to expand its information. A list of functions will appear that you can interact with.
+
+![Remix Functions](intro-to-erc721s/14-functions.png)
+
+The only function you are interested in is the `safeMint` function. Click the drop-down arrow for the function to expand the address field.
+
+![Remix safeMint](intro-to-erc721s/15-safemint.png)
+
+Now, copy your MetaMask address and paste it into this address field. This will send the NFT to your address when the mint function is called. After, hit `transact`.
+
+This will reopen MetaMask and ask you to verify the transaction. Click `Confirm` to mint your NFT.
+
+![Confirm Mint](intro-to-erc721s/16-confirm-mint.png)
+
+Once the transaction has been confirmed, you will see a green checkmark in the terminal at the bottom of the Remix IDE.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "you'll see a green checkmark..." instead of "you will see a green checkmark..."